blocks/header/__burger-menu/header__burger-menu.css

```diff
@@ -0,0 +1,10 @@
+.header__burger-menu
+{
+    width: 22px;
+    height: 16px;
+}
+.burger-menu__icon
+{
+    margin-top: 34px;
+    margin-left: 266px;

```

Comment:
Is this for additional spacing inside header to shift the icon from the left edge?

Probably it's better to have `padding` on some parent element to add spacing around child elements.